packages/server/src/integrations/mongodb.ts

```diff
@@ -6,13 +7,19 @@ import {
 import { IntegrationBase } from "./base/IntegrationBase"
 
 module MongoDBModule {
-  const { MongoClient } = require("mongodb")
+  const { MongoClient, ObjectID } = require("mongodb")
 
   interface MongoDBConfig {
     connectionString: string
     db: string
   }
 
+  interface UpdateDoc {
+    filter: object
+    update: object
+    options: Object
```

Comment:
If the `Object` vs `object` usage above isn't intentional we could make the options an `object` as well. I have always used the upper case variant without much thought, after seeing this I had a look at this great article:
https://2ality.com/2020/01/typing-objects-typescript.html#object-vs.-object-in-typescript

The tldr is `Object` allows for primitives, which we would use defined types for anyway like `string` `number` so we can just use `object`  here and probably in most cases.

Reply:
Good catch! I'd also accidentally brought in an Object class from an aws library, so I removed that import as well.